adafruit_ht16k33/ht16k33.py

```diff
@@ -43,7 +44,7 @@ class HT16K33:
     def __init__(
         self,
         i2c: I2C,
-        address: Union[int, Tuple, List] = 0x70,
+        address: Union[int, List[int], Tuple[int, ...]] = 0x70,
```

Comment:
I'm curious why you have an ellipsis for Tuple, but not List.

Reply:
`List` is a generic that specifies that it can contain any number of the listed type, so `[1]`, [1, 2]`, and `[1, 2, 3]` are all `List[int]`.  `Tuple` on the other hand is a generic that specifies it's elements, so `(1, 2)` is `Tuple[int, int]`.  So `Tuple[int, ...]` specifies a tuple containing `int` _of any size_!